### DISCUSSION TITLE
[07/23] 스프린트 II 회의 시간

### DISCUSSION THREAD

**Discussion by 100pearlcent** [11mLLM/11mLLM#9] [Polls]
## 🗓️ Schedule

전체회의 (1시간) ▷ 팀 별 회의 (약 2시간) ▷ 전체회의 (1시간)

## 📝 Notice

이번 회의는 팀 별 분할 회의를 위해 Zoom으로 진행합니다.
회의 시작 30분 전 디스코드로 회의 링크 공유 예정입니다.

원활한 진행을 위해 회의 전 아래 사항을 미리 셋업하시길 바랍니다.
> (Zoom 계정이 없는 경우) [가입 하러가기](https://us04web.zoom.us) << Click!
> [컴퓨터 기기 사양에 맞는 소프트웨어 다운로드](https://zoom.us/download) << Click!

**Comment by won81** [comment]
아이와 함께 나가야 하는 곳이 있어서 12시 전에 회의에서 나가게 될 것 같습니다.
양해 부탁드리겠습니다.

**Reply by onestone9900** [reply]
저도 일정이 있어서 11시 45분쯤 나가야할 것 같습니다 ㅠ

**Comment by neont21** [comment]
가능하기는 다 가능합니다만 9시 시작을 선호합니다.

**Comment by mel-f-dev** [comment]
9시에 시작하면 9시 반에 나가야할 것 같아요 아예 오후 시간엔 다 가능합니다...!

**Comment by areumseo** [comment]
9시 시작을 희망합니다. 일 때문에 12시까지만 참석하게 될 지도 몰라서요 ^^;;
